metaserver/src/main/scala/scala/meta/languageserver/ScalametaLanguageServer.scala

```diff
@@ -123,14 +129,12 @@ class ScalametaLanguageServer(
   private def onChangedFile(
       path: AbsolutePath
   )(fallback: AbsolutePath => Unit): Unit = {
-    logger.info(s"File $path changed.")
-    val name = path.toNIO.getFileName.toString
-    if (name.endsWith(".semanticdb")) {
-      semanticdbSubscriber.onNext(path)
-    } else if (name.endsWith(".compilerconfig")) {
-      compilerConfigSubscriber.onNext(path)
-    } else {
-      fallback(path)
+    val name = PathIO.extension(path.toNIO)
+    logger.info(s"File $path changed, extension=$name")
+    name match {
+      case "semanticdb" => semanticdbSubscriber.onNext(path)
+      case "compilerconfig" => compilerConfigSubscriber.onNext(path)
```

Comment:
should we also preload all existing `compilerconfig`files at launch, similarly to what we do with `semanticdb` files?